fem/bilininteg.cpp

```diff
@@ -2186,16 +2210,7 @@ void VectorDiffusionIntegrator::AssembleElementMatrix(
    const IntegrationRule *ir = IntRule;
    if (ir == NULL)
    {
-      // integrand is rational function if det(J) is not constant
-      int order = 2 * Trans.OrderGrad(&el); // order of the numerator
-      if (el.Space() == FunctionSpace::rQk)
-      {
-         ir = &RefinedIntRules.Get(el.GetGeomType(), order);
-      }
-      else
-      {
-         ir = &IntRules.Get(el.GetGeomType(), order);
-      }
+      ir = &DiffusionIntegrator::GetRule(el,el);
```

Comment:
@v-dobrev I modified the default `IntegrationRule`in `VectorDiffusionIntegrator::AssembleElementMatrix`, is it what you also had in mind?

Reply:
Yes, it makes sense the for `VectorDiffusionIntegrator` to use the same rules as `DiffusionIntegrator`. Also, don't forget to update the rule in `VectorDiffusionIntegrator::AssembleElementVector` as well.

Since this change may affect users, it will be best to mention it in the CHANGELOG.